### PR TITLE
updates

### PR DESCRIPTION
**Pull request by aelsayed95**

### CODE REVIEW

File: docs/3-restructuredtext-basics.md [hunk 1]
```diff
@@ -48,22 +48,20 @@ section that has some more information and a picture of a sphinx.
...
+Meet Sphinx
+-----------
 ```
 
 ### Image
 
-And now let's add a picture to our new section. You can use any image you like; we chose one we
-found online and downloaded from [unsplash](https://unsplash.com). Save your image in the
-`docs/_static` directory and type under your header:
+And now let's add a picture to our new section. You can use any image you like; we used [flickr](https://www.flickr.com/) and searched for `greek sphinx`. Save your image in the `docs/_static` directory and link it  under your header as follows:
```
Comment (olgarithms) [inline docs/3-restructuredtext-basics.md]: Flickr photos might have copyrights, let's use unsplash.

File: docs/3-restructuredtext-basics.md [hunk 2]
```diff
@@ -48,22 +48,20 @@ section that has some more information and a picture of a sphinx.
 
 Create a new section **above** the "Indices and tables" one:
 
-```
-Workspace
----------
+```rst
+Meet Sphinx
```
Comment (olgarithms) [inline docs/3-restructuredtext-basics.md]: Depending on the photo we can say "the Egyptian Sphinx" or smth

File: docs/3-restructuredtext-basics.md (hunk 1 above)
Comment (olgarithms) [reply docs/3-restructuredtext-basics.md]: How is this? https://unsplash.com/photos/VVzLCuY1xCk

File: docs/3-restructuredtext-basics.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ separate time, so we will focus now on what exists in our `index.rst` file.
 
 ### Heading
 
-```
+```rst
```
Comment (olgarithms) [inline docs/3-restructuredtext-basics.md]: Do we like the way this renders?

File: docs/5-generating-docs.md [hunk 4]
```diff
@@ -29,10 +29,10 @@ need to specify the output path and module path, so from the root of your repo,
 
 ```sh
 # from the project root
-sphinx-apidoc --output-dir docs .
+sphinx-apidoc --output-dir docs src/sphinxy
```
Comment (olgarithms) [inline docs/5-generating-docs.md]:
```suggestion
sphinx-apidoc --output-dir docs src/sphinxy --separate
```

File: docs/5-generating-docs.md [hunk 5]
```diff
@@ -29,10 +29,10 @@ need to specify the output path and module path, so from the root of your repo,
...
 ```
 
-You can see that Sphinx created an `rst` file per module, as well as a `modules.rst` file, which
+You can see that Sphinx created an `rst` file for the `sphinxy` module, as well as a `modules.rst` file, which
```
Comment (olgarithms) [inline docs/5-generating-docs.md]: Here we should say it creates on for the package and two for the `riddle` and `sphinx`

File: docs/3-restructuredtext-basics.md (hunk 1 above)
Comment (aelsayed95) [reply docs/3-restructuredtext-basics.md]: I dont think the Egyptian sphinx has the same story about the riddle? It just guards the city 😅  and splash doesn't have  any photos of greek sphinx :( this is why i went to flickr I think the one I picked didn't have copyright issues but we can check again or reference it? maybe instruct people to reference it idk

**Review by olgarithms — APPROVED** [review]

**Review by olgarithms — APPROVED** [review]

**Pull request merged**